packages/ra-ui-materialui/src/list/ListActions.tsx

```diff
@@ -82,7 +81,19 @@ const ListActions = (props: ListActionsProps) => {
                 )}
             </TopToolbar>
         ),
-        [resource, displayedFilters, filterValues, selectedIds, filters, total] // eslint-disable-line react-hooks/exhaustive-deps
+        [
+            resource,
+            displayedFilters,
+            filterValues,
+            showFilter,
+            filters,
+            total,
+            basePath,
+            className,
+            currentSort,
+            exporter,
+            hasCreate,
+        ] // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
this comment isn't at the right place: the linter still emits a warning for missing props.